docker/kubernetes/ray/ray-master.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ray-master
+  namespace: ray-cluster
+  labels:
+    name: ray-master
+spec:
+  ports:
```

Comment:
We'll need either ports open for the other Redis servers and for the object manager.

Reply:
ok